packages/mco/components/disaster-recovery/applications/failover-and-relocate-modal/dr-policy-selector.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { useDeepCompareMemoize } from '@odf/shared/hooks/deep-compare-memoize';
+import { getName } from '@odf/shared/selectors';
+import { k8sGet } from '@openshift-console/dynamic-plugin-sdk';
+import { TFunction } from 'i18next';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { DRPolicyModel } from '../../../../models';
+import { DRPolicyKind } from '../../../../types';
+import { getErrorMessage } from '../../../../utils';
+import {
+  ACMToDRState,
+  FailoverAndRelocateState,
+  FailoverAndRelocateAction,
+  FailoverAndRelocateType,
+  DRPolicyType,
+} from './reducer';
+
+const getDRPolicyList = async (acmToDRState: ACMToDRState[]) => [
+  ...new Set(
+    acmToDRState.map(
+      (drpcState) => drpcState?.drPolicyControl?.spec?.drPolicyRef?.name
+    )
+  ),
+];
+
+export const DRPolicySelector: React.FC<DRPolicySelectorProps> = (props) => {
+  const { state, dispatch, t } = props;
+
+  // To avoid re-rendering when a new acmToDRState object generated with same props
+  const memoizedAcmToDRState = useDeepCompareMemoize(state.acmToDRState, true);
+
+  const [drPolicies, setDRPolicies] = React.useState<DRPolicyKind[]>([]);
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const setErrorMessage = React.useCallback(
+    (errorMessage: string) => {
+      dispatch({
+        type: FailoverAndRelocateType.SET_ERROR_MESSAGE,
+        payload: {
+          drPolicyErrorMessage: errorMessage,
+        },
+      });
+    },
+    [dispatch]
+  );
+
+  const setSelected = React.useCallback(
+    (selection: DRPolicyType) => {
+      dispatch({
+        type: FailoverAndRelocateType.SET_SELECTED_DR_POLICY,
+        payload: selection,
+      });
+    },
+    [dispatch]
+  );
+
+  React.useEffect(() => {
+    // Fetch DRPolicies using DRPlacementControl
+    const fetchDRPolicies = async (acmToDRState: ACMToDRState[]) => {
+      if (!!acmToDRState.length) {
+        const drPolicyList = await getDRPolicyList(acmToDRState);
```

Comment:
```suggestion
        const drPolicyList = getDRPolicyList(acmToDRState);
```
`getDRPolicyList` is not returning any Promise, we don't need async/await here, right ?